### DISCUSSION TITLE
Pythia Merch (stickers!)

### DISCUSSION THREAD

**Discussion by mgrover1** [ProjectPythia/projectpythia.github.io#138] [Ideas]
During the outreach committee meeting today, the idea of having Project Pythia stickers! These would be a great giveaway item for Twitter giveaways, or even contributor stickers, as an incentive! We could also give these away at conferences, or with people that use our content at workshops and such. What are your thoughts @brian-rose @kmpaul @clyne @dopplershift

**Comment by dcamron** [comment]
+1. We'll be sure to bring this up at next week's meeting too but we hoped to start this process as soon as possible, so we can start offering these as bounty rewards to social media engagement challenges we create. Thanks for the idea Max!
cc @jukent @ktyle

**Comment by clyne** [comment]
Sounds cool. I'm willing to spring for them. Well, assuming reasonable costs :-)

**Comment by brian-rose** [comment]
Sounds great! Since we already have a good logo, I see no reason not to put it on stickers.

**Comment by mgrover1** [comment]
@dopplershift who do you all go through to get the MetPy/Siphon stickers?

**Reply by dopplershift** [reply]
https://www.stickermule.com/

**Comment by mgrover1** [comment]
Unidata has agreed to help with purchasing the stickers for this project! They mentioned wanting to include Unidata somewhere on the sticker... how does everyone feel about this redesigned sticker logo? Is it too busy?
![pythia_logo_with_contributors](https://user-images.githubusercontent.com/26660300/134949396-683c9628-574b-4d0d-b8d3-92479ff49876.png)

**Reply by ktyle** [reply]
Thanks Unidata!! I think, though, that NCAR should probably have the largest "pillar", although that would require different font sizes.

**Reply by dcamron** [reply]
This was mentioned as a "nice to have" from Unidata, but I can try to get clarification on this if we need. I (personally) prefer the sticker either without the extra names/logos, or to find some effective way to have them supplement equally without too much extra going on.

**Comment by brian-rose** [comment]
hmmmm... I hate to be the voice of doom, but I'm uncomfortable with putting the institution names on the sticker like this. The reason is that UAlbany has some pretty strict rules about branding, meaning using the right fonts, colors, etc. It's frustrating, but there's a chance that this sticker could land me in hot water even if Unidata is paying for it.

**Comment by clyne** [comment]
Thanks for your efforts none the less, Max. I think it looks awesome, but would open a can of worms :-( 

If Unidata wants attribution, the best way may be to simply put the institutional logos across the bottom.

**Comment by dcamron** [comment]
@dopplershift got some clarification on my behalf, and looks like it's just fine to leave the sticker design as proposed without any extra names/logos.
